services/ffc/ffc-demo/ffc-demo-web/infra/snd/04/image-policy.yaml

```diff
@@ -11,4 +11,4 @@ spec:
     name: ffc-demo-web
   policy:
     semver:
-      range: '>=0.1.0'
+      range: '>=0.1.0-0'
```

Comment:
@ajaysoni-cap  @ppotturi  can't we override the sem ver at the environment level in the teams config?

Reply:
I'll discuss with @ppotturi this afternoon.